webrtc/RTCPeerConnection-helper.js

```diff
@@ -460,6 +460,39 @@ async function exchangeOfferAndListenToOntrack(t, caller, callee) {
   return ontrackPromise;
 }
 
+// Feature detect whether we can call add transceiver on the target browser.
+// We cannot simply test if the addTransceiver method exist, as Chrome
+// for example would have it implemented but only usable in unified plan,
+// which means it would fail the tests when using default peer connection config
+function canAddTransceiver() {
+  const pc = new RTCPeerConnection();
+  try {
+    pc.addTransceiver('audio');
+    return true;
+  } catch (err) {
+    return false;
+  } finally {
+    pc.close()
+  }
+}
+
+// A dependency agnostic way to add a track track depending
+// on the supported method on a browser.
+//   - If addTransceiver(track) is usable, use that.
+//   - If addTrack(track, mediastream) is available, use that.
+function addTrackOrTransceiver(pc, track, mediaStream) {
+  if (canAddTransceiver()) {
+    return pc.addTransceiver(track);
```

Comment:
We should probably pass the mediaStream to addTransceiver so as to be consistent with addTrack